mitxpro/templatetags/render_bundle_test.py

```diff
@@ -17,83 +16,85 @@
 ]
 
 
-@override_settings(DISABLE_WEBPACK_LOADER_STATS=False)
-class TestRenderBundle(TestCase):
+@pytest.fixture(autouse=True)
+def dont_disable_webpack(settings):
```

Comment:
Might be a good use for indirect fixture parametrization: https://docs.pytest.org/en/latest/example/parametrize.html#apply-indirect-on-particular-arguments

I think you could then pass `pytest.mark.parametrize('disable_webpack', [False], indirect=True)` to `pytestmark`, but I haven't personally tried that

Reply:
It doesn't look like it's working for me. What does the `[False]` do here?